specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/ManagedServerSecurityAlertListByInstance.json

```diff
@@ -0,0 +1,30 @@
+{
+    "parameters": {
+        "subscriptionId": "00000000-1111-2222-3333-444444444444",
+        "resourceGroupName": "securityalert-4799",
+        "managedInstanceName": "securityalert-6440",
+        "api-version": "2017-03-01-preview"
+    },
+    "responses": {
+        "200": {
+            "body": {
+				"value": [
+                    {
+						"id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/securityalert-4799/providers/Microsoft.Sql/managedInstances/securityalert-6440",
+						"name": "Default",
+						"type": "Microsoft.Sql/managedInstances/securityAlertPolicies",
```

Comment:
>"type": [](start = 6, length = 7)

add  location, kind

Reply:
Added only location for server.
"kind" is not relevant.

Reply:
Sorry for the churn, but `location` is spurious in this case. This is a nested proxy resource that does not have "location" in its definition. Location is not required and is not in the response definition in the swagger spec.